package.json

```diff
@@ -31,7 +31,7 @@
   "scripts": {
     "start": "GENERATE_SOURCEMAP=false react-app-rewired start",
     "build": "react-app-rewired build",
-    "test": "react-app-rewired test --coverage a",
+    "test": "jest",
```

Comment:
Probably your test is failing because of this.

Reply:
Aerim could you please give more clear review 🙏. Test is failing, because of what? Thanks
If you mean to add - "test": "react-app-rewired test --coverage a" on 34, it cannot be added as it shows duplicated lines and the line 34 is default Jest in package.json

Reply:
As you can see, you deleted line 34 `"test": "react-app-rewired test --coverage a"` and replaced it with `"test": "jest"`. Don't delete `"test": "react-app-rewired test --coverage a"`.

Reply:
Yes, I have checked it, unfortunately, issue is not there.